http.go

```diff
@@ -100,6 +105,9 @@ type Client struct {
 
 	// NewNotifyClient initializes these services
 	Credentials *NotifyCredentialsService
+
+	// NewLookupClient initializes these services
+	Lookups *LookupsService
```

Comment:
It actually conflicts with the property name for *PhoneNumberPriceService. 

What about? 

```
client.Lookup.LookupPhoneNumbers.Get("+1...")
```

I don't like it much, but can't think anything better. 
Any ideas?